openwisp_controller/config/tests/test_vpn.py

```diff
@@ -630,6 +633,11 @@ def test_update_vpn_server_configuration(self):
                 f'Response status code: 404, VPN Server UUID: {vpn.pk}'
             )
 
+    def test_vpn_peers_changed(self):
+        with catch_signal(vpn_peers_changed) as handler:
+            _, vpn, _ = self._create_wireguard_vpn_template()
+            handler.assert_called_once()
```

Comment:
can you please add a subtest which ensures the signal gets called when a VPN peer is removed?